autocomplete/definitions/namedTypes/tes3lockpick/condition.lua

```diff
@@ -1,5 +1,7 @@
 return {
 	type = "value",
-	description = [[The maximum condition/health of the object.]],
+	description = [[This function is depracted. Use tes3lockpick.maxCondition instead.
+
+The maximum condition/health of the object.]],
```

Comment:
For deprecated fields, I plan on adding `deprecated = true`. Autocomplete can make better use of this, and the docs can then ignore these files to not make things too complicated. No change needed on your end here, but if you want to start using that flag it would save me time in the future.

Reply:
Yeah, I'll use the flag. What do you think about writing down all of the currently available flags somewhere?